src/main/java/de/retest/recheck/printer/AttributeDifferencePrinter.java

```diff
@@ -36,29 +36,29 @@ public AttributeDifferencePrinter( final IdentifyingAttributes attributes, final
 	@Override
 	public String toString( final AttributeDifference difference, final String indent ) {
 		if ( difference.hasElementIdentificationWarning() ) {
-			return indent + format( difference, indent ) + printWarning( difference );
+			return indent + format( difference ) + printWarning( difference );
 		}
-		return indent + format( difference, indent );
+		return indent + format( difference );
 	}
 
-	private String format( final AttributeDifference difference, final String indent ) {
+	private String format( final AttributeDifference difference ) {
 		if ( isExpectedDefault( difference ) ) { // The attribute changed from default to a non-default
-			return printExpectedDefaultDifference( difference, indent );
+			return printExpectedDefaultDifference( difference );
 		}
 		if ( isActualDefault( difference ) ) { // The attribute changed back to default
-			return printActualDefaultDifference( difference, indent );
+			return printActualDefaultDifference( difference );
 		}
-		return printBothDifferences( difference, indent );
+		return printBothDifferences( difference );
 	}
 
 	private boolean isActualDefault( final AttributeDifference difference ) {
 		return defaultProvider.isDefaultValue( attributes, difference.getKey(), difference.getActual() );
 	}
 
-	private String printActualDefaultDifference( final AttributeDifference difference, final String indent ) {
+	private String printActualDefaultDifference( final AttributeDifference difference ) {
 		final String key = highlighter.highlight( difference.getKey(), HighlightType.KEY );
 		final String expected = highlighter.highlight( difference.getExpectedToString(), HighlightType.VALUE_EXPECTED );
-		final String keyExpectedActualFormat = getKeyExpectedActualFormat( indent );
+		final String keyExpectedActualFormat = getKeyExpectedActualFormat( );
```

Comment:
Is your formatter broken? This should be `()`. (*The same for below*)

Reply:
I used Eclipse "refactor method signature". Apparently, that doesn't do proper formatting... Done now.